fix(js): Fix docs for using feedback with the Loader (#12140)
getsentry/sentry-docs: platform-includes/user-feedback/install/javascript.mdx

The User Feedback integration is **already included** in your NPM packages. You don't need to install anything else to use it.

If you're using the Loader Script, you can lazy load the User Feedback integration like this:

```javascript {tabTitle: Loader}
window.sentryOnLoad = function () {
  Sentry.init({
    // add other configuration here
  });

  Sentry.lazyLoadIntegration("feedbackIntegration")
    .then((feedbackIntegration) => {

>       Sentry.addIntegration(feedbackIntegration());

Reviewer: ```suggestion
      Sentry.addIntegration(feedbackIntegration({
      	// configuration options
      }));
```

Reviewer: I just feel like this makes it clearer where to add config, as I had to double check this and wasn't confident

The User Feedback integration is **already included** in your NPM packages. You don't need to install anything else to use it.

If you're using the Loader Script, you can lazy load the User Feedback integration like this:

```javascript {tabTitle: Loader}
window.sentryOnLoad = function () {
  Sentry.init({
    // add other configuration here
  });

  Sentry.lazyLoadIntegration("feedbackIntegration")
    .then((feedbackIntegration) => {
      Sentry.addIntegration(feedbackIntegration({
      	// User Feedback configuration options
      }));
    })
    .catch(() => {
      // this can happen if e.g. a network error occurs,
      // in this case User Feedback will not be enabled
    });
};
```

> If you're using CDN bundles instead of NPM packages, you need to load the User Feedback integration CDN bundle in addition to your browser bundle:

Reviewer 2: "in addition" seems like it should go away. People should pick one CDN script with the combination of integrations that they want to have.

Author: yeah, good point, we call this out in the snippet anyhow!

> If you're using CDN bundles instead of NPM packages, you need to load the User Feedback integration CDN bundle in addition to your browser bundle:

Author: ```suggestion
If you're using CDN bundles instead of NPM packages, you need to use a respective bundle that includes user feedback:
```

Author: @ryan953 like this?
